gubernator/templates/build.html

```diff
@@ -49,7 +49,9 @@
 		% for name, time, text, filename in failures
 			<hr>
 			<h3><a class="anchor" id="{{name|slugify}}" href="#{{name|slugify}}">{{name}}<span class="time"> {{time|duration}}</span></h3></a>
-			<pre class="cmd" onclick="select(this)">{{name | testcmd}}</pre>
+			% if 'junit_runner' not in filename
```

Comment:
It turns out that e2e tests *do* have `classname="Kubernetes e2e suite"`, which is ugly to show in the UI, so I'm just doing it like this instead.